plugins/figma/package.json

```diff
@@ -11,8 +11,8 @@
     "build": "npm run build:plugin && npm run build:ui",
     "build:ui": "vite build -c ./vite.config.ui.ts",
     "build:plugin": "vite build -c ./vite.config.plugin.ts",
-    "types:src": "tsc",
-    "types:node": "tsc -P tsconfig.node.json"
+    "typecheck": "npx tsc",
```

Comment:
Why use `npx tsc`? That package is over 3 years old and quite outdated version of typescript?

Reply:
It doesn't run the package tsc, it runs tsc from `node_modules/.bin`
```
> npx tsc --version
Version 5.5.4
```

But it's a good question, I changed it because `"types:src"` didn't work on my machine and the `tsc` command couldn't be found — but when I'm thinking about it now, that doesn't make any sense, we use the same setup in `packages/cli` after all 🤔 Will double check.

Reply:
I suggest we revert this to `tsc` so that we are sure its using the same version. If we need to change this, do it in a separate PR so that its the same across packages. If anything we should add type checks to other packages that are missing it.

Reply:
I found the problem, it works with just `tsc` if I add TypeScript to the plugins/figma devDependencies